parquet-hadoop/src/main/java/org/apache/parquet/hadoop/InternalParquetRecordReader.java

```diff
@@ -169,13 +169,23 @@ public void initialize(MessageType fileSchema,
     // initialize a ReadContext for this file
     ReadSupport.ReadContext readContext = readSupport.init(new InitContext(
         configuration, toSetMultiMap(fileMetadata), fileSchema));
-    this.requestedSchema = readContext.getRequestedSchema();
```

Comment:
Whether the requestedContext is caseSensitive can com from the readContext. I don't think we need a Hadoop property for this.